docker/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN set -ex \
         apt-utils \
     && useradd -ms /bin/bash -d ${AIRFLOW_USER_HOME} airflow \
     && pip install -U pip
+    && pip install apache-airflow
```

Comment:
why do we need this?
when we install airflow-prometheus-exporter on line 20, we are already installing apache-airflow?

Reply:
Do we need a newer version of Airflow to deal with this error? If so, you can update the version in this file: https://github.com/robinhood/airflow-prometheus-exporter/blob/master/setup.py#L12

Reply:
As explained above, there is an issue with version compatibility between marshmallow and flask-appbuilder if we don't install apache-airflow before airflow-prometheus-exporter.

It's the only way I found as workaround. 

Honestly, I'm not going to investigate that any further.

Reply:
I am not sure what build issue you were seeing earlier but I am not seeing any issues if I run the build locally (without this change). If you want, you can try to remove this line and try again.

Reply:
Without this change:
`ERROR: flask-appbuilder 1.13.1 has requirement marshmallow<2.20,>=2.18.0, but you'll have marshmallow 3.3.0 which is incompatible.`

And now you can see the result in travis.

Reply:
Hello,

I'm interesting in this feature. Could I help with anything to unlock this PR?

Regards